hieradata/class/vagrant/mongo.yaml

```diff
@@ -0,0 +1,49 @@
+---
+mongodb::backup::s3_backups: true
+mongodb::server::replicaset_members:
+  'mongo-1':
+
+mongodb::s3backup::backup::aws_access_key_id: 1234567890 
+mongodb::s3backup::backup::aws_secret_access_key: ABCDEFGHIJKLMNOP1234567890
+mongodb::s3backup::backup::s3_bucket: govuk-mongodb-backup-s3-vagrant
+mongodb::s3backup::backup::standalone: 'True'
```

Comment:
I see that this is configured as a string because when it's interpolated into Python that's correct, but I think it would be less confusing to have a boolean `true/false` YAML value which is then converted into a Python `True/False` later.